packages/flutter/test/cupertino/checkbox_test.dart

```diff
@@ -466,4 +468,101 @@ void main() {
         ),
     );
   });
+
+  testWidgets('Checkbox fill color resolves in enabled/disabled states', (WidgetTester tester) async {
+    const Color activeEnabledFillColor = Color(0xFF000001);
+    const Color activeDisabledFillColor = Color(0xFF000002);
+
+    Color getFillColor(Set<WidgetState> states) {
+      if (states.contains(WidgetState.disabled)) {
+        return activeDisabledFillColor;
+      }
+      return activeEnabledFillColor;
+    }
+
+    final WidgetStateProperty<Color> fillColor =
+      WidgetStateColor.resolveWith(getFillColor);
+
+    Widget buildApp({required bool enabled}) {
+      return CupertinoApp(
+        home: StatefulBuilder(
```

Comment:
Nit: I think you can just set the `home` property of `CupertinoApp` to the `CupertinoCheckbox` directly. Could save some unnecessary lines of code.